frontend/.claude/commands/prd.md

```
@@ -0,0 +1,104 @@
+
+- `MOA-xxx`는 Step 1에서 추출한 스토리 키
+- 태스크 체크리스트 여러 줄은 `\n`으로 연결: `"- [ ] Task1\n- [ ] Task2"`
+- 각 실행 후 출력된 GitHub 이슈 URL을 수집
+
+---
+
+## Phase 4: 완료 출력
+
+모든 이슈 생성 후 아래 형식으로 결과를 출력하세요:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)